week-2/G-conditionals-2/exercise-3.js

```diff
@@ -9,7 +9,15 @@
 */
 
 function calculateGrade(mark) {
-
+  if (mark >= 80) {
+    return 'A';
+  } else if ((mark >= 60) && (mark < 80)) {
+    return 'B';
+  } else if ((mark >= 50) && (mark < 60)) {
```

Comment:
Too many brackets ;)
But good code! Nice spot in the second checks - you need `<` and not `<=`, so well done :)